CS342Application/Views/Member+Views.swift

```diff
@@ -24,6 +25,8 @@ extension Member {
                 Text("Vishnu 👋")
             case .oliverAalami:
                 OliverAalami(navigationPath: navigationPath)
+            case .abhinavAgarwal:
+                AbhinavAgarwal(navigationPath: navigationPath)
```

Comment:
You did not create or commit your `AbhinavAgarwal` view in your Swift Package Target. Please make sure that you commit and add your view to the PR, otherwise the project does not build and you can not use your view in the main application.
Please check out the content of the lecture as well as the other views that are added to the application to draw some inspiration from other solutions.